specification/storage/resource-manager/Microsoft.Storage/preview/2018-03-01-preview/storage.json

```diff
@@ -0,0 +1,2149 @@
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementPolicyName"            
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get management policies successfully.",
+            "schema": {
+              "$ref": "#/definitions/StorageAccountManagementPolicies"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagementPolicies"
+        ],
+        "operationId": "StorageAccounts_CreateOrUpdateManagementPolicies",
+        "description": "Sets the data policy rules associated with the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountSetManagementPolicies": { "$ref": "./examples/StorageAccountSetManagementPolicy.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "name": "accountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the storage account within the specified resource group. Storage account names must be between 3 and 24 characters in length and use numbers and lower-case letters only.",
+            "maxLength": 24,
+            "minLength": 3
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementPolicyName"            
+          },
+          {
+            "name": "properties",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagementPoliciesRulesSetParameter"
+            },
+            "description": "The data policy rules to set to a storage account."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get management policies successfully.",
+            "schema": {
+              "$ref": "#/definitions/StorageAccountManagementPolicies"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagementPolicies"
+        ],
+        "operationId": "StorageAccounts_DeleteManagementPolicies",
+        "description": "Deletes the data policy rules associated with the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountDeleteManagementPolicies": { "$ref": "./examples/StorageAccountDeleteManagementPolicy.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "name": "accountName",
```

Comment:
Could you move accountName property from each method parameter to global parameters list and then just reference it here and everywhere? (same way as you have ResourceGroupName here).

Reply:
For the ResourceGroupName properties, we can't use the global one in specification\common-types\resource-management\v1\types.json, since the "x-ms-parameter-location" are different. 

For StorageAccountName, I have also add a parameter in storage.json for it.